src/view/navigation/routes.ts

```diff
@@ -4,9 +4,12 @@ import FarmerPage from '../pages/Farmer';
 import ExportsPage from '../pages/Exports';
 import LoginPage from '../pages/Login';
 import FarmerSearch from '../pages/FarmerSearch';
-import AddEntryPage from '../components/AddEntry';
-import EditEntryPage from '../components/EditEntry';
-import MilkEntryDetails from '../components/EntryDetails';
+import AddMilkEntryPage from '../components/AddMilkEntry';
+import EditMilkEntryPage from '../components/EditMilkEntry';
+import MilkEntryDetailsPage from '../components/MilkEntryDetails';
```

Comment:
Where are those files you're importing? `AddMilkEntry`, `EditMilkEntry`, ..etc?

Reply:
Added to most recent push